src/containerapp/azext_containerapp/custom.py

```diff
@@ -2162,6 +2162,46 @@ def show_ip_restrictions(cmd, name, resource_group_name):
         return []
 
 
+def set_ingress_sticky_session(cmd, name, resource_group_name, affinity, no_wait=False):
+    _validate_subscription_registered(cmd, CONTAINER_APPS_RP)
```

Comment:
do we need validation here? We would add sticky sessions to a Container App, which means that the sub would already have a validated subscription?

Reply:
no but it is a good idea to have it anyways

Reply:
if you don't need it, remove it. It seems to be making an API call, which is in addition to the call to get the Container App, which in itself is an API call. If you get a Container App for a subscription, that implies that the subscription already has the RP registered right?

Reply:
Other ingress settings are using this validation and therefore i will keep it to follow pattern 
![image](https://user-images.githubusercontent.com/41846928/229983023-16c7bb08-c6ad-4d73-94db-2519f9fc7d03.png)